test/quicksiler-config-client.test.js

```diff
@@ -29,8 +29,8 @@ describe('QuicksilverConfigClient', () => {
       (() => new QuicksilverConfigClient()).should.throw(TypeError);
     });
 
-    // Check API base URL.
-    it('base URL option should be set', () => {
+    // Check QS_CONFIG_PATH has a value.
+    it('QS_CONFIG_PATH in .env should be set', () => {
```

Comment:
Variables from `.env` file are exposed to environment variables. See https://github.com/bkeepers/dotenv